compiler/qsc_frontend/src/lower.rs

```diff
@@ -30,8 +30,12 @@ pub(super) enum Error {
     #[error("invalid attribute arguments: expected {0}")]
     #[diagnostic(code("Qsc.LowerAst.InvalidAttrArgs"))]
     InvalidAttrArgs(String, #[label] Span),
+    #[error("invalid use of the Measurement attribute on a function")]
+    #[diagnostic(code("Qsc.LowerAst.InvalidMeasurementAttrOnFunction"))]
+    InvalidMeasurementAttrOnFunction(#[label] Span),
     #[error("missing callable body")]
     #[diagnostic(code("Qsc.LowerAst.MissingBody"))]
+    #[diagnostic(help("try declaring the callable as an operation"))]
```

Comment:
whoops, looks like the help text got added to the wrong error, should get moved up one.

Reply:
Ahh, good catch!